ci-operator/step-registry/openshift/upgrade/aws/latestinstaller/openshift-upgrade-aws-latestinstaller-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-upgrade-aws-latestinstaller
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - ref: ipi-install-hosted-loki
+    - chain: ipi-aws-pre
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: openshift-e2e-test-capabilities-check
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
```

Comment:
Do we need a new workflow for this, vs. overriding `dependencies` for the default workflow [like this][1]?

[1]: https://github.com/openshift/release/blob/f3cbb682c9174082641a1c2fb17d15045126377a/ci-operator/config/openshift/kubernetes/openshift-kubernetes-release-4.9.yaml#L259-L260

Reply:
We need a new workflow to use different chain

Reply:
Overriding deps would be confusing imo, but possible, yeah.

I'd prefer a separate workflow

Reply:
Overriding `openshift-e2e-test` dependencies won't change imported imagestreams, so openshift-upgrade-aws workflow would still use `stable-initial:installer`